.verdaccio/README.md

```diff
@@ -0,0 +1,27 @@
+# Verdaccio
```

Comment:
This file should be linked in the main readme.md or developer.md to be more easily be found (Cf. Comment from payment user)

Reply:
this README is mostly for when you download the artifact from the PR but I can add a link